Fix typos in docs (#9693)
backstage/backstage: docs/getting-started/configuration.md

That should be it. You can stop your Backstage App. When you start it again and
go to your Backstage portal in your browser, you should have your login prompt!

> To learn more about Authentication in Backstage, here are the following docs you

Reviewer: ```suggestion
To learn more about Authentication in Backstage, here are some docs you
```

the original was kinda correct but using language a bit loosely - the fixed version introduced another oddity though
